src/features/projectsV2/ProjectListControls/styles/ProjectListControls.module.scss

```diff
@@ -20,21 +19,28 @@
   }
 }
 
+.tabsContainer,
 .projectListControlsMobile {
-  height: 28px;
-  justify-content: space-between;
   display: flex;
-  gap: 10px;
-
+  justify-content: space-between;
+  gap: 4px;
+  padding: 0px 10px;
   &.mapModeControls {
     justify-content: flex-end;
   }
 }
 
-.searchFieldAndViewTabsContainer {
-  display: flex;
-  justify-content: space-between;
-  gap: 4px;
+.projectListControlsMobile {
+  height: 26px;
+  gap: 10px;
+}
+
+.marginTop10 {
+  margin-top: 10px;
+}
+
+.marginTop20 {
+  margin-top: 20px;
 }
```

Comment:
1. Still not sure why we need 2 different margins. 
2. Also, while using utility classes is fine, using a mixture of utility classes and other classes can cause confusion. In this scenario, I feel it would be better to use a specific class for impersonation mode.